dist/less/order-service.less

```diff
@@ -170,17 +175,24 @@
       }
     }
   }
+  .review-message {
+    padding-left: 10px;
+  }
+  .review-status {
+    display: flex;
+    font-size: 16px;
+
+    .pficon {
+      margin-top: 3px;
+    }
+  }
 }
 
-.wizard-pf-contents {
+.order-service-wizard-step.wizard-pf-main {
   @media (min-width: @screen-sm-min) {
-    display: flex;
-    min-height: 100%;
-    padding-bottom: (@grid-gutter-width / 2);
-    padding-top: (@grid-gutter-width / 2);
-    .scroll-shadows-horizontal-covers();
-    _::-webkit-full-page-media, _:future, :root & { // only target Safari
-      background: none;
-    }
+    height: calc(@order-service-page-height + @grid-gutter-width);
```

Comment:
I know we've had some debate on this and I understand why height/max-height are added, but I agree with @rhamilto. As a user, only being able to use a small portion of my screen to fill out what could be a complicated form is frustrating. We don't control what parameters come back from the brokers, and the form will get longer when we add parameters in #231. I feel like the minor inconvenience of the dialog resizing and the buttons moving is the lesser evil, particularly when our wizard only has a few steps.